pkg/injection/namespace/bootstrapperconfig/bootstrapperconfig.go

```diff
@@ -0,0 +1,325 @@
+package bootstrapperconfig
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"strings"
+
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/enrichment/endpoint"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/ca"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/curl"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/pmc"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	dtclient "github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
+	"github.com/Dynatrace/dynatrace-operator/pkg/consts"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/activegate/capability"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/connectioninfo"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/processmoduleconfigsecret"
+	"github.com/Dynatrace/dynatrace-operator/pkg/injection/namespace/mapper"
+	"github.com/Dynatrace/dynatrace-operator/pkg/injection/startup"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/conditions"
+	k8slabels "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/labels"
+	k8ssecret "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/secret"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// BootstrapperInitGenerator manages the bootstrapper init secret generation for the user namespaces.
+type BootstrapperInitGenerator struct {
+	client        client.Client
+	dtClient      dtclient.Client
+	apiReader     client.Reader
+	namespace     string
+	secretName    string
+	canWatchNodes bool
+}
+
+type nodeInfo struct {
+	imNodes map[string]string
+	nodes   []corev1.Node
+}
+
+func NewBootstrapperInitGenerator(client client.Client, apiReader client.Reader, dtClient dtclient.Client, namespace string) *BootstrapperInitGenerator {
```

Comment:
```suggestion
func NewSecretGenerator(client client.Client, apiReader client.Reader, dtClient dtclient.Client, namespace string) *BootstrapperInitGenerator {
```

so when you import the package: `bootstrapperconfig .NewSecretGenerator`

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4515/commits/3ec51819b9c3e36f2094612a55f30fa969f45796